source/extensions/filters/http/cache/simple_http_cache/BUILD

```diff
@@ -29,10 +27,6 @@ envoy_cc_extension(
         "//source/common/protobuf",
         "//source/extensions/filters/http/cache:http_cache_lib",
         "//source/extensions/filters/http/common:pass_through_filter_lib",
+        "@envoy_api//envoy/extensions/filters/http/cache/v3alpha/simple_http_cache:pkg_cc_proto",
```

Comment:
ci is trying to remove this - not sure why - afaict its needed

Reply:
im guessing its because its not used in the header file - beyond my code knowledge tbh - either way ci is pretty determined  it doesnt want to allow it - the cache filter itself has a similar import and i think it is needed

Reply:
Maybe add a header include, I think there is precedent, can't remember right now but a comment explaining is fine.